lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Yes, the state will never be fixed. I put it in the payment plugin is for consistency with other plugins such as CyberSource. If we put it in a control plugin, that will work as well; but we would need some if else logic to identify it is a `Orbital` payment and then cancel it. Seems me that the cancelation logic would be in two places separately. For some payment gateway, you need to look at the plugin itself; while for others, you need to look at the control plugin.